apps/ledger-live-desktop/src/renderer/components/RecoverRestore/index.tsx

```diff
@@ -53,7 +55,20 @@ const RecoverRestore = () => {
         }
       },
       error: (error: Error) => {
-        setError(error);
+        if (
+          (error instanceof TransportStatusError && error.message.includes("0x6d06")) ||
+          error instanceof DeviceNotOnboarded
+        ) {
+          setState({
+            isOnboarded: false,
+            isInRecoveryMode: false,
+            seedPhraseType: SeedPhraseType.TwentyFour,
```

Comment:
I'm not sure to understand, why do we set a 24 words type if the device is not onbarded?